hack/nudging/container_digest.sh

```diff
@@ -1,8 +1,8 @@
 # Do not remove empty lines, they are there to reduce conflicts
-export OPERATOR_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-rhel9-operator@sha256:304440234657a05d3fde85a780f23b29c80fcb7089b38c1b86b24dd799665c54'
+export OPERATOR_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-rhel9-operator@sha256:e059192784283385687011b9fa4bb87094783d4530aa66b79168777666a6e2a0'
```

Comment:
This one has changed since your PR :-) 

```suggestion
export OPERATOR_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-rhel9-operator@sha256:d8ff9f17ebbf1190632571aab60a7c9fbc7373acec63876e178ae09292805645'
```